qemu/tests/block_resize.py

```diff
@@ -1,14 +1,16 @@
 import logging
 import re
 
-from autotest.client.shared import error
```

Comment:
I don't think it's a good solution to handle the different between autotest and avocado, 
as more and more difference between them and we won't develop on autotest framework any more.

we can create a branch to tp-qemu for autotest framework.

cc @xutian